pkg/ui/src/redux/nodes.ts

```diff
@@ -211,19 +211,22 @@ export function nodeCapacityStats(n: NodeStatus$Properties): CapacityStats {
   };
 }
 
-export function getDisplayName(node: NodeStatus$Properties) {
-  return `${node.desc.address.address_field} (n${node.desc.node_id})`;
+export function getDisplayName(node: NodeStatus$Properties, decommissioned?: boolean) {
```

Comment:
perhaps pass in the liveness status here?  that would give us moderately more type help.